docs/user-guide/sources/redis-source.md

```diff
@@ -18,9 +20,9 @@ spec:
 ```
 
 Please see [API](https://github.com/numaproj/numaflow/blob/main/docs/APIs.md#redisstreamssource) for details on how to optionally do the following:
-- Define TLS
-- Define username/password
-- Connect to Redis Sentinel 
```

Comment:
not really related, but I noticed the dashes didn't effectively turn into bullets, but the asterisks do